test/test_automl.py

```diff
@@ -151,6 +151,30 @@ def test_ensemble(self):
         '''The main flaml automl API'''
         automl.fit(X_train=X_train, y_train=y_train, **settings)
 
+    def test_preprocess(self):
```

Comment:
should the test also include an explicit case when X is numpy array since the issue we fixed is for numpy array? or it is converted later into numpy array by the code itself?

Reply:
It's converted by the stacking ensemble code. See https://github.com/microsoft/FLAML/issues/130#issuecomment-887771442